git_remote_s3/common.py

```diff
@@ -5,24 +5,24 @@
 import re
 
 
-def parse_git_url(url: str) -> tuple[str, str, str]:
+def parse_git_url(url: str) -> tuple[str, str, str, str]:
     """Parses the elements in an s3:// remote origin URI
 
     Args:
         url (str): the URI to parse
 
     Returns:
-        tuple[str, str, str]: prefix, bucket and profile extracted from the URI or
-        None, None, None if the URI is invalid
+        tuple[str, str, str, str]: uri scheme, prefix, bucket and profile extracted
+        from the URI or None, None, None, None if the URI is invalid
     """
     if url is None:
-        return None, None, None
-    m = re.match(r"s3://([^@]+@)?([a-z0-9][a-z0-9\.-]{2,62})/?(.+)?", url)
-    if m is None or len(m.groups()) != 3:
-        return None, None, None
-    profile, bucket, prefix = m.groups()
+        return None, None, None, None
+    m = re.match(r"(s3.*)://([^@]+@)?([a-z0-9][a-z0-9\.-]{2,62})/?(.+)?", url)
```

Comment:
change to accept `s3` and `s3+zip` schemas only using  `(s3|s3\+zip)` instead of `(s3.*)`